rear_rider_device/leds_proc.py

```diff
@@ -137,34 +113,50 @@ async def on_strobe_off(self):
         # self.writeline(
         #     'strobe_off_ack'
         # )
-        await self._join_strobe_thread()
+        self._leds_effects_loop_ctx.set_effects([])
         # self.writeline(
         #     'strobe_off_ok'
         # )
```

Comment:
mind removing the commented out code around this?

Reply:
no

Reply:
I don't mind